src/components/textareaInput/textareaInput.module.css

```diff
@@ -0,0 +1,21 @@
+.inputWrapper {
+	padding-bottom: var(--size-s);
+}
+
+.label {
+	display: block;
+}
+
+.input {
+	min-width: 300px;
+	min-height: 150px;
+	height: var(--size-m);
+	border-radius: var(--border-radius-semi-rounded);
+	border: 2px solid var(--color-yellow);
+	padding: var(--size-xs);
+	font: unset;
+}
+
+.input:focus {
+	outline-color: var(--color-pale-pink);
```

Comment:
same as the other style comment I left, I think it might be nice to also add this style on hover